### PR TITLE
What's New for engine 2.9

### PR DESCRIPTION
**Pull request by micheles**
See https://github.com/gem/oq-engine/blob/doc/doc/whats-new.md

### CODE REVIEW

**Comment by CatalinaYepes** [comment]
The link for the complete list of changes in the event based calculator at the beginning is broken

**Comment by daniviga** [comment]
@CatalinaYepes the link will work when we'll branch for the release

File: doc/whats-new.md [hunk 1]
```diff
@@ -1,157 +1,289 @@
...
+The command `oq extract` has now a signature consistent with `oq export`
+and has grown the ability to connect to a remote engine server.
+
+The `oq dump` command has been extended so that it is possible to extract
+a single calculation, if so desired.
+
+IT
+---
+
+We have specialized packages for clusters. The packages meant for the controller
```
Comment (daniviga) [inline doc/whats-new.md]: ..., also for Ubuntu.

RHEL was already there since couple of versions ago

File: doc/whats-new.md [hunk 2]
```diff
@@ -1,157 +1,289 @@
...
+The version of numpy officially supported by the engine is 1.11 however
+we made some work to ensure that the tests run also with the most recent
+versions. In the near future we will probably migrate to numpy 1.14.
+
+Since we added a field with the process ID in the job table in order to
+implement the abort functionality, upgrading to engine 2.9 requires a
+database migration if you have an older version of the engine and you want
+to keep your old calculations. You will just need to say yes the first time you
+run a calculation, when the engine will ask you to upgrade the database.
+The migration is immediate and safe.
```
Comment (daniviga) [inline doc/whats-new.md]: I would add "Docker containers can be used now to deploy a multi-node installation of the Engine on a cluster".

File: doc/whats-new.md (hunk 1 above)
Comment (micheles) [reply doc/whats-new.md]: Done

File: doc/whats-new.md (hunk 2 above)
Comment (micheles) [reply doc/whats-new.md]: Done

**Pull request merged**